style/js/README.md

```diff
@@ -1,3 +1,190 @@
+JavaScript Style Guides
+=====
+
+Guides for programing JavaScript in style
+
+General
+---
+* Use [JSHint](http://jshint.com/)
+* Always use 'use strict'
+
+Parens and Braces
+---
+* Leave one space between `if` and `(`.
+
+```javascript
+if (something) {
+  //some work here;
+}
+```
+
+* Prefer this form for multiline if statements.
+
+```javascript
+if (something) {
+  // do stuff
+  // even more stuff
+} else {
+  // some other stuff
+  // real stuff
+}
+```
+
+* On if statements, whenever it makes sense, you can omit the `{}`.
```

Comment:
Why do we want this?

I can see some disadvantages:

1. We have more than one way of doing the same thing (if statements) and thus have to think which one to use. This is something where I believe no thinking should be involved.
2. If it grows to more than one line per branch, we need to change it, which is putting the "pain" forward to the next guy.
3. It fails jslint or jshint, one of those, I believe.

On the other hand, the only advantage I see is that we spare a few characters.